packages/core/src/App/Containers/AcuityDownloadModal/acuity-download-modal.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import RootStore from 'Stores/index';
+import { Button, Modal, Text, Icon } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { getUrlBase } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+import 'Sass/app/modules/acuity-download.scss';
+
+type TAcuityDownloadModal = {
+    is_acuity_modal_open: boolean;
+    setIsAcuityModalOpen: (value: boolean) => void;
+};
+
+const AcuityDownloadModal = ({ is_acuity_modal_open, setIsAcuityModalOpen }: TAcuityDownloadModal) => {
+    const closeModal = () => setIsAcuityModalOpen(false);
+
+    const openDownloadLink = () => {
+        window.open('https://deriv.link/3WhYxq1', '_blank', 'noopener,noreferrer');
```

Comment:
https://caniuse.com/?search=window.open Could be a problem on safari based browser

Reply:
Let's have a look into this in a separate PR, it has been widely used in several different notifications.